spec/spec_helper.rb

```diff
@@ -80,30 +80,10 @@ def windows2016?
   is_win2k16
 end
 
-def chef_lt_12_5?
-  Gem::Version.new(Chef::VERSION) < Gem::Version.new('12.5')
-end
-
-def chef_gte_12?
-  Chef::VERSION.split('.').first.to_i >= 12
-end
-
-def chef_gte_12_5?
-  Gem::Version.new(Chef::VERSION) >= Gem::Version.new('12.5')
-end
-
```

Comment:
We removed this here so according to us this condition is not working in https://github.com/chef/knife-windows/pull/475/files#diff-1c44cb7c409c533aa88dc2f6ea3c1802R174.  I guess we should verify this.